pages/style/color/overview.md

```diff
@@ -117,8 +117,9 @@ USWDS helps teams choose accessible colors with a color grade system. Let’s lo
 
 ### Magic number
 We call the difference in grade between any two colors the _magic number_. Magic numbers have important contrast implications:
-- **A magic number of `40`** or higher achieves Section 508 AA Large contrast (For example: between `gray 90` and `indigo warm 50v`).
-- **A magic number of `50`** or higher achieves Section 508 AA contrast (For example: between `gray 90` and `red 40`).
+- **A magic number of `40`** or higher achieves WCAG 2.0 AA Large Text contrast (For example: between `gray 90` and `indigo warm 50v`).
+- **A magic number of `50`** or higher achieves WCAG 2.0 AA contrast or AAA Large Text contrast(For example: between `gray 90` and `red 40`).
```

Comment:
- Add space between `contrast` and `(`
- remove the `For` in `For example`; don't uppercase the `e`